cmd/release-controller/sync_verify_prow.go

```diff
@@ -56,6 +56,7 @@ func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName str
 	}, map[string]string{
 		releaseAnnotationSource: fmt.Sprintf("%s/%s", release.Source.Namespace, release.Source.Name),
 	})
+	pj.Spec.Cluster = c.prowJobCluster
```

Comment:
Why would we ever tolerate this being different than what's configured in the base job in config?

Reply:
The base job doesn't have any config and shouldn't need to know about CI details like this

Reply:
Why doesn't it? Why shouldn't it? Why is it appropriate for the release controller to e.g. publish a job to the app.ci cluster? IMO the control for where the job should run is not here.

Reply:
Because where the jobs run is our problem, how the jobs look like is arts problem.

Reply:
Really? Explain to me why the release controller should get to choose which build cluster this runs on when there is only one valid place for it -- build01.

Reply:
The idea is not to leave it a choice but in the contrary enforce something. We do not write these jobs, but we do control the deployment of the release-controller or not?